apps/nextra/components/diagrams/use-draw.ts

```diff
@@ -6,7 +6,7 @@ export function useDraw(paths = {}, name = "") {
     defaultLocale: string;
   };
 
-  if (!Object.hasOwn(paths, defaultLocale)) {
+  if (!Object.prototype.hasOwnProperty.call(paths, defaultLocale)) {
```

Comment:
hmm interesting, how many browsers wouldn't support `hasOwn`

This answer on stack overflow points that this is preferred _https://stackoverflow.com/questions/69561596/object-hasown-vs-object-prototype-hasownproperty_

Seems like most modern browsers support it?